libraries/helpers.rb

```diff
@@ -113,8 +113,6 @@ def product_lookup(product, version = '0.0.0')
       elsif (product == 'push-server') && (v < Mixlib::Versioning.parse('2.0.0'))
         data['package-name'] = 'opscode-push-jobs-server'
         data['ctl-command'] = 'opscode-push-jobs-server-ctl'
-      elsif (product == 'push-client') && (v < Mixlib::Versioning.parse('2.0.0'))
```

Comment:
It could be some older releases of Push Jobs Client were not published to PackageCloud (cause they are old) BUT the `1.3.0` value is based on looking in the stable repo of Artifactory which serves as our system of record and feeds external systems like PackageCloud and Omnitruck:

![artifactory_chef_s_artifactory_instance____repository_browser](https://cloud.githubusercontent.com/assets/6164/10984064/ec053304-83e4-11e5-8717-a3a9ef4516af.png)